src/app/components/bitcoin-transaction-item/bitcoin-transaction-item.layout.tsx

```diff
@@ -10,6 +10,7 @@ interface BitcoinTransactionItemLayoutProps {
   txStatus: JSX.Element;
   txTitle: JSX.Element;
   txValue: JSX.Element;
+  increaseFeeButton?: JSX.Element;
```

Comment:
On naming, this is really just putting _something_ below the caption. Is it possible we'd wanna put something else there, too?

What if this was just a `belowCaption` prop, or something that names the place where X is put, rather than X itself.